build/commands/lib/depotTools.js

```diff
@@ -133,27 +133,19 @@ function installDepotTools(options = config.defaultOptions) {
       }
     }
 
-    const ninjaLogCfgPath = path.join(config.depotToolsDir, 'ninjalog.cfg')
-    if (!fs.existsSync(ninjaLogCfgPath)) {
-      // Create a ninja config to prevent autoninja from calling "cipd
-      // auth-info" each time. See for details:
-      // https://chromium.googlesource.com/chromium/tools/depot_tools/+/main/ninjalog.README.md
-      const ninjaLogCfgConfig = {
-        'is-googler': false,
-        'version': 3,
-        'countdown': 10,
-        'opt-in': false
-      }
-      fs.writeFileSync(ninjaLogCfgPath, JSON.stringify(ninjaLogCfgConfig))
-    }
-
```

Comment:
I think we should keep both methods until https://chromium-review.googlesource.com/c/chromium/tools/depot_tools/+/5669094 is merged.